platforms/minestom/src/main/java/com/dfsek/terra/minestom/TerraMinestomExample.java

```diff
@@ -0,0 +1,158 @@
+package com.dfsek.terra.minestom;
+
+import com.dfsek.terra.minestom.api.filter.ChunkFilter;
+import com.dfsek.terra.minestom.api.filter.EvenChunkFilter;
+import com.dfsek.terra.minestom.api.filter.NoFeaturesFilter;
+import com.dfsek.terra.minestom.api.filter.NoTerrainFilter;
+import com.dfsek.terra.minestom.api.filter.SpecificChunkFilter;
+import com.dfsek.terra.minestom.api.filter.XFilter;
+import com.dfsek.terra.minestom.api.filter.ZFilter;
+import com.dfsek.terra.minestom.world.TerraMinestomWorld;
+import com.dfsek.terra.minestom.world.TerraMinestomWorldBuilder;
+
+import net.kyori.adventure.text.Component;
+import net.minestom.server.MinecraftServer;
+import net.minestom.server.command.builder.Command;
+import net.minestom.server.command.builder.arguments.ArgumentLiteral;
+import net.minestom.server.command.builder.arguments.number.ArgumentInteger;
+import net.minestom.server.coordinate.Pos;
+import net.minestom.server.entity.GameMode;
+import net.minestom.server.event.player.AsyncPlayerConfigurationEvent;
+import net.minestom.server.event.player.PlayerSpawnEvent;
+import net.minestom.server.instance.Instance;
+import net.minestom.server.instance.LightingChunk;
+import org.jetbrains.annotations.Nullable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicInteger;
+
+
+public class TerraMinestomExample {
```

Comment:
Why is this file committed to the `main` source set?

Reply:
Do you think it would be good to create a seperate module for the example? I think that might the best option

Currently I have it in there because it makes it easy to try out.

What do you think?

Reply:
it should absolutely be a separate module.